Configuration/Geometry/scripts/dict2023Geometry.py

```diff
@@ -353,6 +353,78 @@
             'from Geometry.EcalMapping.EcalMappingRecord_cfi import *',
         ],
         "era" : "run2_HE_2017, run2_HF_2017, run2_HCAL_2017, run3_HB, phase2_hcal, phase2_hgcal",
+    },
+    "C3" : {

```

Comment:
@slava77 it's acceptable if @bsunanda wants to add the necessary geometry info for C3 without making a workflow for it right away. (So in general, adding items is okay, but editing anything that's *currently used* in a workflow should be accompanied by rerunning the script to update any affected cff files.)

But yes, I would appreciate keeping the README up to date, i.e. C3 corresponds to HGCal v8 + Phase2 HCAL.